sandbox/settings.py

```diff
@@ -578,6 +578,11 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         environ_prefix=None,
     )
 
+    # Features
```

Comment:
The features flags code is related this PR https://github.com/openfun/richie/pull/1912/files

I think it will be better that you remove this code from you "wishlist feature" commits and you rebase your branch from the branch "rlecellier/feature_flags" :)

This way, I can do my comment about this subject in the right PR